README.rst

```diff
@@ -105,6 +103,17 @@ news and updates.
 .. _its Web site: https://beets.io/
 .. _@b33ts: https://twitter.com/b33ts/
 
+Contact
+-------
+* Encountered a bug you'd like to report or have an idea for a new feature? Check our our `issue tracker`_! If your issue or feature hasn't already been reported, please `open a new ticket`_ and we'll be in touch with you shortly.
+* Need help/support, would like to start a discussion, or would just like to introduce yourself to the team? Check out our `forums`_!
+* If you are a developer or a prospective developer looking for some guidance, we have a `gitter chat`_. Feel free to stop by anytime!
```

Comment:
I'm hoping gitter can become more of a common-place dev-focused chat room in favor of any previously suggested irc (or whatever else) channels.

Reply:
Did you see my [note on discourse](https://discourse.beets.io/t/forming-a-beets-core-team/639/15?u=jyrik) about gitter dropping mobile support? May make it less accessible than other options if you're willing to consider other avenues.

Reply:
I'll leave the gitter stuff in here as I assume we'll continue to use it until an alternative is decided upon.

Reply:
Seems the gitter is likely to be abandoned now. Should we even advertise it? Or just direct people to the discourse?

Reply:
I don't have a strong feeling either way—there's no particular harm in leaving the think for now, even if the Gitter is not long for this world.